package.json

```diff
@@ -62,7 +62,6 @@
     "diff": "^4.0.1",
     "eslint-plugin-no-unsanitized": "^3.0.2",
     "eslint-plugin-prefer-object-spread": "^1.2.1",
-    "flat": "^4.1.1",
```

Comment:
does it safe to remove it?

Reply:
Yes, dashboard has the dependency of flat.